.idea/Mathball.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
These xml and iml files are not needed, also package-lock.json is not needed.
So, redo your commits after removing them.

Also, please update the `README.md` and `src/index.js` with the links to the updated modules

Reply:
To remove these files, I will have to delete the existing pull request, and create a new one.

Reply:
You can just remove your `iml` and `xml` files and commit and push again, and they should be gone.
Don't forget to make changesd to `README.md` and `src/index.js` while at it.
Also squash your commits into one, if multiple commits done.